Storage/rest/basic.h

```diff
@@ -123,7 +123,8 @@ struct Basic {
       input.entry.InitializeOwnKey();
       if (!Exists(input.field[current::storage::sfinae::GetKey(input.entry)])) {
         input.field.Add(input.entry);
-        return Response(current::ToString(current::storage::sfinae::GetKey(input.entry)), HTTPResponseCode.Created);
+        return Response(current::ToString(current::storage::sfinae::GetKey(input.entry)),
```

Comment:
> Or {"success":true, "url":"https://..."} ?

Probably. Also a separate short representation of the created resource would also be useful, though this could likely not be fully restful (just want to save a GET request).
`{"success":true, "url":"https://...", "data": {"foo": "bar"}}`